src/plugins/vdo_stats.c

```diff
@@ -67,9 +68,9 @@ get_stat_val_double (GHashTable *stats, const gchar *key, gdouble *val) {
 static void add_write_ampl_r_stats (GHashTable *stats) {
     gint64 bios_meta_write, bios_out_write, bios_in_write;
 
-    if (! get_stat_val64 (stats, "bios_meta_write", &bios_meta_write) ||
-        ! get_stat_val64 (stats, "bios_out_write", &bios_out_write) ||
-        ! get_stat_val64 (stats, "bios_in_write", &bios_in_write))
+    if (! get_stat_val64 (stats, "biosMetaWrite", &bios_meta_write) ||
```

Comment:
Hmm, it's not stated in libblockdev API docs anywhere, however we have an explicit list of key names in UDisks API: https://github.com/storaged-project/udisks/blob/cd8314b638c57b84a3d2855c81fa14c5e4ad2fe2/modules/lvm2/data/org.freedesktop.UDisks2.lvm2.xml#L904-L1011

So either we create just another translation table in UDisks, or we carry it in libblockdev. Kinda reminds me the smart plugin where we had to keep the libatasmart key names.

Reply:
I saw the udisks documentation and if it was up to me, I'd be ok with changing the "output". VDO was never part of the upstream kernel before so I'd just pretend the past didn't happen and say this is the new "API" and we ignore everything else. I am already ignoring the existence of `kvdo` with this change.

Reply:
Good point about the first public version. After all, VDO guys need to sign up for some output format stability themselves (unless dm messages are intentionally not part of the guarantees).

Reply:
The output should be stable, but I wouldn't be surprised if it changes in the future. I hope it will at least always be a valid YAML so the parsing works.